### PR TITLE
add an example related to panorama

### PR DESCRIPTION
**Pull request by hopepdm**

### CODE REVIEW

**Comment by Mugen87** [comment]
Can you please elaborate a bit the purpose of this example?

File: examples/webgl_panorama_shaderMaterial.html [hunk 1]
```diff
@@ -0,0 +1,166 @@
...
+            controls.rotateSpeed = - 0.25;
+
+            var params = {
+                alp: 0,
+                opacity: 1
+            }
+
+            uniforms = {
+                map1: {
+                    type: 't',
```
Comment (WestLangley) [inline examples/webgl_panorama_shaderMaterial.html:57]: FYI - This `type` property is no longer needed.

Comment (hopepdm) [reply examples/webgl_panorama_shaderMaterial.html:57]: thank you for reminding me

File: examples/webgl_panorama_shaderMaterial.html [hunk 2]
```diff
@@ -0,0 +1,166 @@
...
+            var container = document.getElementById( 'container' );
+
+            renderer = new THREE.WebGLRenderer();
+            renderer.setPixelRatio( window.devicePixelRatio );
+            renderer.setSize( window.innerWidth, window.innerHeight );
+            container.appendChild( renderer.domElement );
+
+            scene = new THREE.Scene();
+
+            camera = new THREE.PerspectiveCamera( 90, window.innerWidth / window.innerHeight, 0.1, 1000 );
```
Comment (WestLangley) [inline examples/webgl_panorama_shaderMaterial.html:41]: FYI - `.fov` of `90` is way too big, causing significant distortion -- especially in the "cabin" scene. Use `50` instead.

**Comment by WestLangley** [comment]
temporary live link: https://raw.githack.com/hopepdm/three.js/dev/examples/webgl_panorama_shaderMaterial.html

**Comment by hopepdm** [comment]
> Can you please elaborate a bit the purpose of this example?

It's easier to create a panorama through shadermaterial, and it's more natural to transform the panorama

**Comment by Mugen87** [comment]
Sorry, but I don't think `three.js` requires an official example that demonstrates crossfading between two skyboxes.

I suggest you share your code as a resource in the [forum](https://discourse.threejs.org/).

**Comment by WestLangley** [comment]
>I suggest you share your code as a resource in the forum.

@hopepdm I also think that would be nice. :-)

**Pull request closed without merging**